Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -345,6 +346,9 @@ def bump_formula_pr
       alias_rename.map! { |a| formula.tap.alias_dir/a }
     end
 
+    ohai "brew bump-python-resources #{formula.name}"
+    PyPI.update_python_resources! formula, silent: true, odie_on_error: false unless args.dry_run? && !args.write?
```

Comment:
```suggestion
    PyPI.update_python_resources! formula, silent: true, odie_on_error: false if !args.dry_run? && !args.write?
```

More readable that way IMO

Reply:
Quick little truth table:

| `dry_run?` | `write?` | run?
|---|---|---|
|✅|✅|✅|
|✅|❌|❌|
|❌|✅|✅ (n/a)|
|❌|❌|✅|

I think the best way to fit this is:
```ruby
unless args.dry_run? && !args.write?
```

To use `if`, it would need to be:
```ruby
if args.dry_run? && args.write? || !args.dry_run?
```

which I think is less clear

Reply:
Hmm, this looks nice in table, but not so nice in the code. Anyway, if you think it's fine this way, so be it.

Reply:
Does adding parentheses look any better?

```ruby
PyPI.update_python_resources!(formula, silent: true, odie_on_error: false) unless args.dry_run? && !args.write?
```

I think it might make it more clear. The `false unless ` throws me off a little, I think

Reply:
While it may not look great, if you read it it makes sense I think:

"run `update_python_resources` unless you passed dry run and didn't pass write"